nginx.tmpl

```diff
@@ -22,6 +22,13 @@ map $http_x_forwarded_proto $proxy_x_forwarded_proto {
   ''      $scheme;
 }
 
+# If we received a HTTP-CloudFront-Forwarded-Proto, pass it through; otherwise,
+# pass along the original value.
+map $http_cloudfront_forwarded_proto $proxy_x_forwarded_proto {
+  default $proxy_x_forwarded_proto;
```

Comment:
This bit confuses me; `$http_cloudfront_forwarded_proto` would make sense as we're forwarding the header; I'm reading this as if it sets the header to it's current/existing value?

Also confusing: what happens if both `Forwarded-Proto` variations are sent(?)